### DISCUSSION TITLE
Trouble Populating QuickTime:GPSCoordinates Field

### DISCUSSION THREAD

**Discussion by teddimagg** [photostructure/exiftool-vendored.js#150] [Q&A] [answered]
Hello

I am currently working on a Node.js project that requires the addition of location metadata to `.mov` and `.mp4` video files using ExifTool. All of these video files are QuickTime formatted. Despite my code executing without errors, I'm facing a recurring issue where the `QuickTime:GPSCoordinates` field (equivalent to `com.apple.quicktime.location.ISO6709`) remains unpopulated in the final output.

Here is the relevant part of my code:

```javascript
const exifObject = {
  'CreateDate': creationDate,
  'DateTimeOriginal': creationDate,
  'QuickTime:CreationDate': creationDate,
  'QuickTime:GPSCoordinates': iso6709Coordinates
};
await exiftool.write(fileName, exifObject, ['-overwrite_original']);

```
The _iso6709Coordinates_ are in the ISO 6709:2008 format.

I am not sure why the QuickTime:GPSCoordinates field isn't populating as expected. Is there something that I may be missing or doing incorrectly in my code? Any guidance on this matter would be greatly appreciated.

Thank you in advance for your assistance,

**Comment by mceachen** [comment, marked as answer]
Looks like you want the `Keys` group instead of `QuickTime`? https://exiftool.org/forum/index.php?topic=10081.0

Also: make sure you check out
https://exiftool.org/TagNames/QuickTime.html

(You may need to specify latitude and longitude separately)

**Reply by teddimagg** [reply]
Thank you, it worked :)
